### PR TITLE
Standard integrations that emit custom metrics

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Move "Standard integrations that emit custom metrics" under billing section
- Update outdated links
- Add info on billable count

### Motivation
FAQ audit

### Preview link
https://docs-staging.datadoghq.com/ruth/standard-custom-metrics/account_management/billing/custom_metrics/

### Additional Notes
Integrations PR in integrations-core

### CODE REVIEW

File: content/en/account_management/billing/custom_metrics.md [hunk 1]
```diff
@@ -1,22 +1,57 @@
...
+  - /integrations/faq/what-standard-integrations-emit-custom-metrics/
 ---
 
 ## Overview
 
-Custom metrics refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack. Some [standard integrations][1] can potentially emit custom metrics.
+In general, [custom metrics][1] refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack.
 
-Pro plans include 100 custom metrics per host and Enterprise plans include 200 custom metrics per host. This number is averaged across all paid hosts. Additional custom metrics packages can be purchased.
+* Billable count is based on average number of `custom metrics / hour` for the month.
```
Comment (l0k0ms) [inline content/en/account_management/billing/custom_metrics.md]:
```suggestion
* **Billable count is based on average number of `custom metrics / hour` for the month**.
```

File: content/en/account_management/billing/custom_metrics.md [hunk 2]
```diff
@@ -1,22 +1,57 @@
...
+* Pro plans include 100 custom metrics per host.
+* Enterprise plans include 200 custom metrics per host.
+* The number of metrics is averaged across all paid hosts.
+* Additional custom metrics packages can be purchased.
```
Comment (l0k0ms) [inline content/en/account_management/billing/custom_metrics.md]: Should we link to sales@datadoghq.com here?

File: content/en/account_management/billing/custom_metrics.md [hunk 3]
```diff
@@ -1,22 +1,57 @@
...
 
 Contact [Sales][2] or your [Customer Success][3] Manager to discuss custom metrics for your account.
 
+### Standard integrations
+Some standard integrations can potentially emit custom metrics.
+
+Integrations limited to 350 metrics by default that emit custom metrics:
```
Comment (l0k0ms) [inline content/en/account_management/billing/custom_metrics.md]:
```suggestion
Integrations limited to 350 custom metrics by default:
```

File: content/en/account_management/billing/custom_metrics.md [hunk 4]
```diff
@@ -1,22 +1,57 @@
...
+
+* [ActiveMQ XML][4]
+* [Go-Expvar][5]
+
+Integrations with no default limit that emit custom metrics:
```
Comment (l0k0ms) [inline content/en/account_management/billing/custom_metrics.md]:
```suggestion
Integrations with no default limit over the amount of custom metrics emitted: 
```

File: content/en/account_management/billing/custom_metrics.md (hunk 1 above)
Comment (ruthnaebeck) [reply content/en/account_management/billing/custom_metrics.md]: I don't like bolding entire sentences.

File: content/en/account_management/billing/custom_metrics.md (hunk 2 above)
Comment (ruthnaebeck) [reply content/en/account_management/billing/custom_metrics.md]: Sales is linked in the line below.

File: content/en/account_management/billing/custom_metrics.md [hunk 5]
```diff
@@ -1,22 +1,57 @@
...
+* Billable count is based on average number of `custom metrics / hour` for the month.
+* Pro plans include 100 custom metrics per host.
+* Enterprise plans include 200 custom metrics per host.
+* The number of metrics is averaged across all paid hosts.
+* Additional custom metrics packages can be purchased.
 
 Contact [Sales][2] or your [Customer Success][3] Manager to discuss custom metrics for your account.
 
+### Standard integrations
+Some standard integrations can potentially emit custom metrics.
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/custom_metrics.md]:
```suggestion
The following standard integrations can potentially emit custom metrics.
```

**Pull request merged**